run_tests.py

```diff
@@ -7,6 +7,6 @@
 
 workdir = os.getcwd()
 
-runner = TestRunnerBase(test_suite_name, test_data_files_info=os.path.join(os.path.dirname(__file__),"share", "test_data_files.txt"))
+runner = TestRunnerBase(test_suite_name)
```

Comment:
I think I had the original one in case you do not run it from the repo dir, but it's not an issue anymore